index.bs

```diff
@@ -2842,6 +2846,10 @@ Object">WebVTT Internal Node Objects</a> also have an ordered list of class name
 their <dfn lt="WebVTT Node Object's applicable language">applicable language</dfn>, which is to be
 interpreted as a BCP 47 language tag. [[!BCP47]]</p>
 
+<p class=note>User agents will add a language tag as the <a lt="WebVTT Node Object's applicable
```

Comment:
I understand the goal of this note, but it seems like it isn't worded correctly, since we want to encourage user agents to supply a well-formed and valid language tag, but not require them to check the language tag. I would probably say:

```
User agents must add a language tag as the applicable language. While language tag should be a valid language tag [[!BCP47]], the user agent is not required to check the structure or contents of the language tag to ensure that this is so.
```

Reply:
Except notes are non-normative and shouldn't use normative keywords (should, required).